new-lamassu-admin/src/pages/Wallet/helper.js

```diff
@@ -111,12 +126,13 @@ const getElements = (cryptoCurrencies, accounts, onChange, wizard = false) => {
       name: 'zeroConfLimit',
       size: 'sm',
       stripe: true,
-      view: it => it,
+      view: getZeroConfLimit,
       input: NumberInput,
       width: 190 - widthAdjust,
       inputProps: {
         decimalPlaces: 0
-      }
+      },
+      editable: row => row.id !== 'ETH'
```

Comment:
I'm not sure if this change on `editableTable` is needed to accomodate for this usage, mostly to avoid messing around with an already problematic component. Check with Rafael and in case the change is positive, confirm if this change breaks any other usage of `editableTable`